src/Symfony/Component/Console/Formatter/NullOutputFormatter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Formatter;
+
+/**
+ * @author Tien Xuan Vo <tien.xuan.vo@gmail.com>
+ */
+class NullOutputFormatter implements OutputFormatterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function format(?string $message): void
+    {
+        // do nothing
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStyle(string $name): OutputFormatterStyleInterface
+    {
+        // to comply with the interface we must return a OutputFormatterStyleInterface
+        return new NullOutputFormatterStyle();
```

Comment:
Should we define a `$formatter` property? So that `getStyle` method doesn't create a new one each calling time.

```php
/**
 * {@inheritdoc}
 */
public function getStyle(string $name): OutputFormatterStyleInterface
{
    if ($this->formatter) {
        return $this->formatter;
    }
    // to comply with the interface we must return a OutputFormatterStyleInterface
    return $this->formatter = new NullOutputFormatterStyle();
}
```

Reply:
I'm not sure about this. Can someone help me on this? cc @Tobion

Reply:
sounds good to me. 
@tienvx can you make `final` all classes introduced here also?